src/wallet/wallet.h

```diff
@@ -135,6 +135,9 @@ enum WalletFlags : uint64_t {
     // wallet flags in the upper section (> 1 << 31) will lead to not opening the wallet if flag is unknown
```

Comment:
Should this say `upper section (> 0 << 31)` ?

Reply:
Not my problem :p